### PR TITLE
Self-review

### PR DESCRIPTION
**Pull request by sergiogval**
This pull request is just to make an exercise of reviewcode.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-Will be delivered in the following steps of the project.
+Click here to see the live demo.
```
Comment (sergiogval) [inline README.md:19]: I would suggest you kindly add the link to the live demo. An empty hyperlink may confuse a reader on your Readme file

File: index.html [hunk 2]
```diff
@@ -152,8 +152,49 @@ <h3>Skills</h3>
...
+        <a href="#"><img src="images/ic_twitter.png" alt="twitter"></a>
+      </li>
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_linkedin.png" alt="linkedin"></a>
+      </li>
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_medium.png" alt="Medium"></a>
+      </li>
+      <li class="socialicon">
+        <a href="#"><img src="images/ic_angellist.png" alt="Angelist"></a>
```
Comment (sergiogval) [inline index.html:194]: Great work so far. I only will suggest that the icons for the social media have a real link to your pages. That will avoid broken links and improve the UX.

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,3 +1,4 @@
 {
-  "deepscan.enable": true
+  "deepscan.enable": true,
+  "liveServer.settings.port": 1987,
```
Comment (sergiogval) [inline .vscode/settings.json:3]: Good that you change the port for live server functionality. That improves security. I will only recommend that you add this settings to the .gitignore file in order to do not show them in your public repository.

**Review by sergiogval — COMMENTED** [review]
Hi @sergiogval 👋🏾 ,

Good job so far! :partying_face: 
You made and excellent job on:

Follow best practices for HTML & CSS :octocat:

That said, there are some issues that you still need to work on to go to the next project but you are almost there!
Required Changes ♻️

_Check the comments bellow_

Feel free to reach me if you have any questions.

File: .vscode/settings.json (hunk 3 above)
Comment (rotimiazeez) [inline .vscode/settings.json:3]: - [x] Nice review you have here👍👍 Kindly add emoji's to make the review friendlier😉

File: README.md (hunk 1 above)
Comment (rotimiazeez) [inline README.md:19]: - [x] Good job with this review and nice job adding the why👍👍 Kindly add emoji's to make the review friendlier😉

File: index.html (hunk 2 above)
Comment (rotimiazeez) [reply index.html:194]: - [x] Good job with this review and nice job adding the why👍👍 as stated earlier kindly add emoji's to make the review friendlier😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @sergiogval 👋, 
## Good job so far!👏👏

### There are some issues that you still need to work on to go to the next project but you are almost there!😉
> Your review was great👍, kindly implement the suggested changes and make the review even better✔🎉

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @sergiogval , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sergiogval** [comment]
Close PR and delete branch.

**Pull request closed without merging**